### DISCUSSION TITLE
doesn't refresh when filter is enabled

### DISCUSSION THREAD

**Discussion by imed-ghomari** [RafaelGB/obsidian-db-folder#355] [Ideas]
**Describe the bug**
When a filter is enabled, and the user changes the select option to something that should be hidden by the filter. The db doesn't refresh automatically. 
related to #300 and #305 

**Screenshots**

https://user-images.githubusercontent.com/38974541/189478815-b5b6e3f2-2e6c-4181-a335-2a7f625fa161.mov

**Desktop (please complete the following information):**
- macOS 12.3.1
- Obsidian 0.15.9

**Comment by RafaelGB** [comment]
The refresh of the filter is a heavy operation. It involves calling dataview again and reloading all the information in the table


Maybe you are looking for a custom search filter. The search filters is not that heavy and are called with every change

**Reply by imed-ghomari** [reply]
Okay, good to know. It's not that bad, I can always toggle the filter on and off.
